entities/racetrack_entity.py

```diff
@@ -0,0 +1,58 @@
+import pygame
+
+
+class RaceTrack():
+    def __init__(self, accelaration, max_speed, friction, spaceship, screen):
+        self.accelaration = accelaration
+        self.spaceship = spaceship
+        self.max_speed = max_speed
+        self.friction = friction
+        self.visible_colliders = []  # Storing colliders as Rect objects
+        self.speed = 2  # Initial speed

```

Comment:
```suggestion
        self.speed = INITIAL_SPEED # Initial speed

```

Pode ser melhor definir isso como uma constante lá em cima